style.css

```
@@ -1,3 +1,361 @@
+  width: fit-content;
+  padding: 1rem;
+  background-color: #E9E6E6;
+  grid-area: i;
+}
+
+.show-summary-container {
+  background-color: #E9E6E6;
+  margin: 1rem;
+  padding: 1rem;
+  grid-area: s;
+}
+
+.show-summary-text {
+  width: 18rem;
+  text-align: justify;
+}
+
+
+.show-info-container {
+  display: flex;
+  flex-direction: column;
+  justify-self: center;
+  width: 18rem;
+  padding: 1rem;
+  background-color: #E9E6E6;
+  grid-area: b;
+}
+
+.episode-name-num-holder {
+  border: 0.1rem black solid;
+  border-radius: 0.3rem;
+  display: flex;
+  justify-content: center;
+  width: 20rem;
+}
+
+.episode-name-num-text {
+  font-family: "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
+    "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
+}
+
+.episode-image-container {
+  padding: 1rem;
+}
+
+.episode-summary-text {
+  font-size: small;
+  padding: 0 1rem 1rem 1rem;
+  font-size: medium;
+}
+
+
+.show-card {
+  display: flex;
+  flex-direction: column;
+  border: 0.1rem #84d8ec solid;
+  background-color: #D1EBF1;
+  width: 18rem;
+  border-radius: 0.3rem;
+  justify-content: center;
+  /* align-items: center; */
+  padding: 1rem;
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-template-rows: auto;
+  grid-gap: 2rem;
+  grid-template-areas:
+    "n"
+    "i"
+    "s"
+    "b";
+}
+
+
+#footer {
+  display: flex;
+  flex-direction: column;
+}
+
+#source {
+  display: flex;
+  justify-content: center;
+}
+
+
+@media screen and (min-width: 480px) {
+
+  #root {
+  display: flex;
+  justify-content: center;
+  background-color: #8fe4df;
+  color: black;
+  padding: 1rem;
+}
+
+  .nav-bar-holder {
+      padding: 1rem;
+      background-color: #E9E6E6;
+
+  }
+
+  .nav-bar {
+    display: flex;
+    flex-direction: row;
+    height: 2rem;
+    display: block; 
+  }
+
+
+  #root {
+    display: flex;
+    justify-content: center;
+    background-color: #8fe4df;
+    color: black;
+    padding: 1rem;
+  }
+  
+  
+  .show-cards-container {
+    background-color: #E9E6E6;
+    border: 0.05rem #db9f9f solid;
+    border-radius: 0.5rem;
+
+    display: flex;
+    flex-direction: column;
+    width: 85rem;
+    justify-items: center;
+    justify-content: center;
+    padding: 2rem;
+  }
+
+  .episode-cards-container {
+    background-color: #E9E6E6;
+    border: 0.05rem #db9f9f solid;
+    border-radius: 0.5rem;
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+    justify-content: center;
+    gap: 3rem;
+    width: 85rem;
+    padding: 2rem;
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(20rem, 1fr));
+  }
+  
+  .show-summary-text {
+    width: auto;
+    padding: 1rem;
+  }
+
+.episode-card {
+  display: flex;
+  flex-direction: column;
+  border: 0.1rem black solid;
+  background-color:#D1EBF1;;
+  width: 23rem;
+  border-radius: 0.3rem;
+  align-items: center;
+  padding: 1rem;
+}
+
+    .show-card {
+    padding-bottom: 1rem;
+    gap: 0;
+    width: 80rem;
```

The problem in your rem measurements. Better to use %
```suggestion
    width: 100%;
```